.github/workflows/build.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         run: npm run test
       - name: Public Api Check
         run: npm run public-api:check
-      - name: Upload Demo Artifact
+      - name: Upload Storybook Artifact
         uses: actions/upload-artifact@v3
         with:
-          name: demo
-          path: dist/clr-demo/
+          name: demo # todo: change this to storybook, update demo.yml also
```

Comment:
Reminder to change this before merging. It will break the PR demo deployment for this PR, but after it's merged, it will work again.